server/src/main/java/eu/fasten/server/FastenServer.java

```diff
@@ -236,7 +243,9 @@ private void waitForInterruption(List<FastenServerPlugin> plugins) {
             return new FastenKafkaPlugin(consumerProperties, producerProperties, k, skipOffsets,
```

Comment:
It seems that `enableKafka` flag is not passed here.

Reply:
I added another constructor which defaults `enableKafka` to false. I indeed need this for test code. I wanted to be able to isolate parts of the FastenKafkaPlugin to test, without starting Kafka consumer/producer. Since these are initialized in the constructor, I couldn't find an easier workaround.

Reply:
Shouldn't the default value be `true`? if the flag is not passed in `FastenServer.java`?

Reply:
Yes this is the case, see [here](https://github.com/fasten-project/fasten/blob/abe026545bab72ed54290047525279dbcea70471/server/src/main/java/eu/fasten/server/plugins/kafka/FastenKafkaPlugin.java#L111)